src/lib.rs

```diff
@@ -193,14 +193,14 @@ mod tests {
         libc_print::libc_println!("# rand::distributions::DistString");
 
         distribution_test(false, |count, len| {
-            use rand::distributions::DistString;
+            use rand::distr::SampleString;
```

Comment:
Renamed, see the changelog https://github.com/rust-random/rand/blob/master/CHANGELOG.md